app/pages/lab/mobile/mobile-section-container.jsx

```diff
@@ -45,14 +45,47 @@ function workflowQuestionHasOneOrLessImages({ task }) {
   return validation;
 }
 
+function drawingToolTypeIsValid({ task }) {
+  let validationBool = false;
+  if (task.tools[0]) {
+    validationBool = task.tools[0].type === 'rectangle';
+  }
+  return convertBooleanToValidation(validationBool);
+}
+
+function drawingTaskHasOneTool({ task }) {
+  return convertBooleanToValidation(task.tools.length === 1);
+}
+
+function drawingTaskHasNoSubtasks({ task }) {
+  const toolHasSubtasks = task.tools.reduce((hasSubtasks, tool) => {
+    return hasSubtasks || tool.details.length > 0;
+  }, false);
+
+  return convertBooleanToValidation(!toolHasSubtasks);
+}
+
 const validatorFns = {
-  taskQuestionNotTooLong,
-  taskFeedbackDisabled,
-  taskHasTwoAnswers,
-  workflowFlipbookDisabled,
-  workflowHasSingleTask,
-  workflowNotTooManyShortcuts,
-  workflowQuestionHasOneOrLessImages
+  single: {
+    taskQuestionNotTooLong,
+    taskFeedbackDisabled,
+    taskHasTwoAnswers,
+    workflowFlipbookDisabled,
+    workflowHasSingleTask,
+    workflowNotTooManyShortcuts,
+    workflowQuestionHasOneOrLessImages
+  },
+  drawing: {
+    taskFeedbackDisabled,
+    workflowFlipbookDisabled,
+    workflowHasSingleTask,
+    drawingToolTypeIsValid,
+    drawingTaskHasOneTool,
+    drawingTaskHasNoSubtasks
```

Comment:
Although we're deprecating the shortcut task, should this have a check `workflowDoesntContainShortcuts` or something similar? I don't think a shortcut will appear on the app with a drawing task.